shared/src/vyxal/Elements.scala

```diff
@@ -2525,6 +2537,17 @@ object Elements:
         val temp =
           ListHelpers.transliterate(ListHelpers.makeIterable(a), b, c).mkString
         if VNum.NumRegex.matches(temp) then VNum(temp) else temp
+      case (a: VObject, b: String, c) => MiscHelpers.setObjectMember(a, b, c)
+      case (a: VObject, b: VList, c) =>
+        var obj = a.copy()
```

Comment:
Is it necessary to copy it right here if `setObjectMember` won't mutate it anyway?

Reply:
when you put it that way, no